tests/kubectl_test.go

```diff
@@ -79,7 +79,7 @@ var _ = Describe("[rfe_id:3423][vendor:cnv-qe@redhat.com][level:component]oc/kub
 		// Verify one of the wide column output field
 		Expect(resultFields[len(resultFields)-verifyPos]).To(Equal(expectedData))
 	},
-		table.Entry("[test_id:3468]virtualmachine", "get", "vm", "wide", []string{"NAME", "AGE", "RUNNING", "VOLUME", "CREATED"}, 1, "true"),
+		table.Entry("[test_id:3468]virtualmachine", "get", "vm", "wide", []string{"NAME", "AGE", "VOLUME", "STRATEGY", "CREATED"}, 1, "true"),
```

Comment:
STRATEGY isn't a column that is registered. i'm not sure how this would pass.

Reply:
@davidvossel I missed that one. Will fix it and the description as well. Thanks!

Reply:
@davidvossel Done